controllers/utils/bloom_filter.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import "math"
+
+// CalculateBloomFilterSize Accepts the size of the IPFS storage in bytes
+// and returns the bloom filter size to be used in bytes.
+//
+// Computations are done with values based on this issue:
+// https://github.com/redhat-et/ipfs-operator/issues/35#issue-1320941289
+func CalculateBloomFilterSize(ipfsStorage uint64) uint64 {
+	// 256KiB
+	const blockSize = 262144
+	// number of blocks
+	var n, m uint64
+	// false-negative rate, 1 / 1000
+	var p = 0.001
+
+	n = ipfsStorage / blockSize
+
+	// formula based on bloom filter calculator:
+	// https://hur.st/bloomfilter
+	m = uint64(math.Ceil((float64(n) * math.Log(p)) / math.Log(1/math.Pow(2, math.Log(2)))))
+	return m
```

Comment:
**Important**: this is in `bits` and should be converted into bytes for use in the bloom filter config value. (I don't see this happening even though comments say it returns bytes.